### PR TITLE
[16.0][ADD] account_multi_foreign_vat

### PR DESCRIPTION
**Pull request by ArnauCForgeFlow**

### CODE REVIEW

File: account_multi_foreign_vat/models/partner.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+from odoo import api, models
+
+
+class AccountFiscalPosition(models.Model):
+
+    _inherit = "account.fiscal.position"
```
Comment (LoisRForgeFlow) [inline account_multi_foreign_vat/models/partner.py]: You are inheriting twice account.fiscal.position in tow different files. Can you move this extension to the file with the name of the model?

File: account_multi_foreign_vat/models/account_fiscal_position.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+                    raise ValidationError(
+                        _(
+                            "The country detected for this foreign VAT "
+                            "number does not match the one set on this fiscal position."
+                        )
+                    ) from e
+                if not checked_country_code and not record.country_id:
+                    raise ValidationError(
+                        _("The foreign VAT number is not correct.")
+                    ) from e
```
Comment (LoisRForgeFlow) [inline account_multi_foreign_vat/models/account_fiscal_position.py:36]: Is this the only change done to the standard method? https://github.com/odoo/odoo/blob/7d678cc7f4c9e00d78d4120f703e90bd282c1a51/addons/base_vat/models/account_fiscal_position.py

File: account_multi_foreign_vat/models/partner.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+                if record.state_ids:
+                    similar_fpos_domain.append(
+                        ("state_ids", "in", record.state_ids.ids)
+                    )
+
+                similar_fpos_count = self.env["account.fiscal.position"].search_count(
+                    similar_fpos_domain
+                )
+                if similar_fpos_count:
+                    pass
```
Comment (LoisRForgeFlow) [inline account_multi_foreign_vat/models/partner.py]: at the end this code is just skipping the full check, isn't it? Why dont you just return pass?

File: account_multi_foreign_vat/README.rst [hunk 4]
```diff
@@ -0,0 +1,41 @@
+======================================
+Account Multi Foreign VAT
```
Comment (LoisRForgeFlow) [inline account_multi_foreign_vat/README.rst]: README files in OCA are autogenerated from fragments, see https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

**Review by LoisRForgeFlow — COMMENTED** [review]
Some comments

File: account_multi_foreign_vat/models/partner.py (hunk 1 above)
Comment (ArnauCForgeFlow) [reply account_multi_foreign_vat/models/partner.py]: Sure, my bad

File: account_multi_foreign_vat/README.rst (hunk 4 above)
Comment (ArnauCForgeFlow) [reply account_multi_foreign_vat/README.rst]: Okay, I didn't know

File: account_multi_foreign_vat/models/account_fiscal_position.py (hunk 2 above)
Comment (ArnauCForgeFlow) [reply account_multi_foreign_vat/models/account_fiscal_position.py:36]: Yes

File: account_multi_foreign_vat/models/partner.py (hunk 3 above)
Comment (ArnauCForgeFlow) [reply account_multi_foreign_vat/models/partner.py]: Yes, you are right

**Comment by dreispt** [comment]
Are requested changes done @LoisRForgeFlow ?

**Comment by JordiMForgeFlow** [comment]
@dreispt could we reopen this one?

**Pull request closed without merging**